include/glow/Importer/CommonOperatorLoader.h

```diff
@@ -675,8 +675,18 @@ class CommonOperatorLoader : public ProtobufLoader {
     const std::string &opName = loadOperatorName(op);
     NodeValue in;
     ASSIGN_VALUE_OR_RETURN_ERR(in, getNodeValueByName(op.input(0)));
-    unsigned_t k;
-    ASSIGN_VALUE_OR_RETURN_ERR(k, loadInt(dict["k"]));
+    RETURN_ERR_IF_NOT(op.input_size() <= 2, "Maximum number of inputs is 2.");
+    unsigned_t k = 0;
+    if (op.input_size() > 1) {
+      Constant *kConst = getConstantByNameOrNull(op.input(1));
+      RETURN_ERR_IF_NOT(kConst, "K is not Constant.");
+      RETURN_ERR_IF_NOT(kConst->getElementType() == ElemKind::Int64ITy,
+                        "k input must be of type Int64");
```

Comment:
"K" for consistent naming

Reply:
updated code to have consistent error message.